Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -1748,11 +1748,11 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
 
     configuration._processDisplayName = extension().webProcessDisplayName();
 
+    // FIXME: <https://webkit.org/b/263286> Consider allowing the background page to throttle or be suspended.
     auto *preferences = configuration.preferences;
-#if PLATFORM(MAC)
-    preferences._domTimersThrottlingEnabled = NO;
-#endif
+    preferences._hiddenPageDOMTimerThrottlingEnabled = NO;
     preferences._pageVisibilityBasedProcessSuppressionEnabled = NO;
+    preferences.inactiveSchedulingPolicy = WKInactiveSchedulingPolicyNone;
```

Comment:
This is a new property in iOS 17/macOS 14, but I guess it's always present internally even if we build for down levels.

Reply:
Yep.